internal/collector/nginxossreceiver/metadata.yaml

```diff
@@ -7,7 +7,21 @@ status:
     beta: [metrics]
   distributions: [contrib]
   codeowners:
-    active: [olli-holmala, oliveromahony, apgralG, dhurley, craigell]
+    active: [olli-holmala, oliveromahony, apgralG, dhurley, craigell, sean-breen]
+
+resource_attributes:
+  resource.id:
```

Comment:
Don't think you need to add resource.id here since that is added outside of the receiver